contracts/truefi2/deprecated/LoanToken2Deprecated.sol

```diff
@@ -0,0 +1,471 @@
+// SPDX-License-Identifier: MIT
```

Comment:
Can we have this inherit from LoanToken2 and override decimals?

Reply:
I wanted it to match exactly on chain version but tbh it might just require to write some integration tests